specs/ecosystem/ecosystem-multiarch.spec

```diff
@@ -0,0 +1,117 @@
+PIPELINES-33
+# Verify ecosystem E2E spec
+
+Pre condition:
+  * Validate Operator should be installed
+
+## jib-maven pipelinerun: PIPELINES-33-TC01
+Tags: linux/amd64, ecosystem, non-admin, jib-maven, sanity
+Component: Pipelines
+Level: Integration
+Type: Functional
+Importance: Critical
+
+Steps:
+  * Verify ServiceAccount "pipeline" exist
+  * Create
+      |S.NO|resource_dir                                          |
+      |----|------------------------------------------------------|
+      |1   |testdata/ecosystem/pipelines/jib-maven.yaml      |
```

Comment:
align `|` so that we have a nice table :)

Reply:
Done